src/js/components/Skeleton/stories/GridContainer.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+
+import { Box, Button, Card, Grid, Image, Text} from 'grommet';
+import { FormAdd } from 'grommet-icons';
+
+const labels = [];
+for (let i = 0; i < 10; i += 1) labels.push(`Item ${i}`);
+
+const skeleton = { message: { start: "Loading", end: 'Content Loaded' } };
+
+const Item = ({ title, ...rest }) => (
+  <Card
+    width="531px"
+    pad="medium"
+    gap="large"
+    round="medium"
+    flex="grow"
+    {...rest}
+  >
+    <Box direction="row" gap="large" justify="between" align="center">
+      <Box direction="row" gap="medium">
+        <Box width="96px" height="96px" background="brand" round="small">
```

Comment:
Any way to avoid px value and use tshirt size instead?

Reply:
I replaced most of the pixel sizes with tshirt sizes. I only kept one of the card widths as pixels